src/App.js

```diff
@@ -1,9 +1,106 @@
+import { useState } from "react";
+import './style.css'; //이번 과제는 컴포넌트 분리가 어려워서 css를 사용했습니다 ㅠㅠ
+import './background.js';
+import Clock from './clock.js';
 
 function App() {
+  const [toDo, inputToDo] = useState("");
+  const [toDos, setToDos] = useState([]);
+  const [doneToDos, movetoDos] = useState([]); 
+
+  //inputToDo를 통해 입력받음
+  const onChange = (event) => inputToDo(event.target.value);
+
+
+  const submitInput = (event) => {
+      event.preventDefault();
+      if (toDo === "") {
+        return;
+      }
+      setToDos((currentArray) => [toDo, ...currentArray]); //입력받은 걸 배열에 넣음
```

Comment:
이런 식으로 작성하면 코드를 더 간결하게 짤 수 있겠죠?

```suggestion
      if (toDo) {
      setToDos((currentArray) => [toDo, ...currentArray]); //입력받은 걸 배열에 넣음
    }
```